### DISCUSSION TITLE
spatie translatable plugin disable button?

### DISCUSSION THREAD

**Discussion by esdriver04** [filamentphp/filament#15045] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

Hello, I want to close the button used in the spatie translatable package on the edit page.

**Comment by enessvg** [comment, marked as answer]
Hello,
```php

Actions\LocaleSwitcher::make()->disabled(),

```
```php
->disabled
````
will help you

**Reply by esdriver04** [reply]
Thank you!
